### PR TITLE
[DPP-1233] Update UM/UP docs

### PR DESCRIPTION
**Pull request by pbatko-da**
changelog_begin
changelog_end

### CODE REVIEW

File: docs/source/app-dev/services.rst [hunk 1]
```diff
@@ -207,7 +207,7 @@ Utility Services
 Party Management Service
 ========================
 
-Use the **party management service** to allocate parties on the ledger and retrieve information about allocated parties.
+Use the **party management service** to allocate parties on the ledger, update participant-local party propertiesgi and retrieve information about allocated parties.
```
Comment (carrielaben-da) [inline docs/source/app-dev/services.rst:210]:
```suggestion
Use the **party management service** to allocate parties on the ledger, update participant and local party properties and retrieve information about allocated parties.
```

File: ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/object_meta.proto [hunk 2]
```diff
@@ -15,7 +15,7 @@ option csharp_namespace = "Com.Daml.Ledger.Api.V1.Admin";
 // See https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/generated.proto#L640
 message ObjectMeta {
 
-  // An opaque value populated by a participant server which represents the internal version of the resource
+  // An opaque non-empty value populated by a participant server which represents the internal version of the resource
```
Comment (carrielaben-da) [inline ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/object_meta.proto:18]:
```suggestion
  // An opaque, non-empty value, populated by a participant server, which represents the internal version of the resource
```

**Review by carrielaben-da — APPROVED** [review]
Minor edits

File: docs/source/app-dev/services.rst (hunk 1 above)
Comment (mziolekda) [reply docs/source/app-dev/services.rst:210]: Pawel did mean "participant-local", but that's probably too much of a developer slang. Maybe `party properties local to participant`?

Comment (carrielaben-da) [reply docs/source/app-dev/services.rst:210]: "party properties local to the participant" would work I think.

Comment (pbatko-da) [reply docs/source/app-dev/services.rst:210]: Thanks, changing to `update party properties local to the participant`

File: ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/object_meta.proto (hunk 2 above)
Comment (pbatko-da) [reply ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/object_meta.proto]: Ok, applied

**Review by mziolekda — APPROVED** [review]

**Pull request merged**